tests/invoice_utils/conftest.py

```diff
@@ -33,6 +33,11 @@ def f(name: str) -> str:
     return f
 
 
+@pytest.fixture
+def test_basic_rules(read_text):
```

Comment:
`test_*` is the convention for test names, not fixture names. please rename to `basic_rules`.